.github/workflows/requirements-upgrade.yml

```diff
@@ -0,0 +1,58 @@
+name: Upgrade Python Requirements
```

Comment:
Do we still need this, with the new org-wide workflow template?

Reply:
Removed.